cat.go

```diff
@@ -31,19 +30,17 @@ func Cat(paths ...string) Stream {
 	return Stream{
 		r:     readcloser{Reader: io.MultiReader(readers...), Closer: closers},
 		stage: "cat",
-		err:   errors.ErrorOrNil(),
+		err:   merr,
 	}
-
 }
 
 type multicloser []io.Closer
 
-func (mc multicloser) Close() error {
-	var errors *multierror.Error
+func (mc multicloser) Close() (merr error) {
```

Comment:
Can you please not use named return parameter with naked return here and keep the local `errors` variable. Just change the type.

Reply:
I can switch away from the named return, but we have to change the local variable name, otherwise it conflicts with the `errors` package which has the `Join` function.